internal/users/users.go

```diff
@@ -161,6 +165,116 @@ func computeGroupOperation(newGroupsInfo []GroupInfo, currentLocalGroups []strin
 	return groupsToAdd, groupsToRemove
 }
 
+// CleanUserFromSystemGroups removes the user from all system groups.
+func CleanUserFromSystemGroups(user string, args ...Option) (err error) {
```

Comment:
I think we should use local groups instead of system groups (some can be local and not be handled by the system). The existing implementation always mentioned local and not system groups.